processing/src/main/java/org/apache/druid/query/aggregation/any/DoubleAnyAggregator.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.aggregation.any;
+
+import org.apache.druid.query.aggregation.Aggregator;
+import org.apache.druid.segment.BaseDoubleColumnValueSelector;
+
+public class DoubleAnyAggregator implements Aggregator
+{
+  private final BaseDoubleColumnValueSelector valueSelector;
+
+  private Double foundValue;
+
+  public DoubleAnyAggregator(BaseDoubleColumnValueSelector valueSelector)
+  {
+    this.valueSelector = valueSelector;
+    foundValue = null;
+  }
+
+  @Override
+  public void aggregate()
+  {
+    if (foundValue == null && !valueSelector.isNull()) {
+      foundValue = valueSelector.getDouble();
+    }
+  }
+
+  @Override
+  public Object get()
+  {
+    return foundValue;
+  }
+
+  @Override
+  public float getFloat()
+  {
+    return foundValue.floatValue();
+  }
+
+  @Override
+  public long getLong()
+  {
+    return foundValue.longValue();
+  }
+
+  @Override
+  public double getDouble()
+  {
+    return foundValue;
+  }
+
+  @Override
+  public void close()
+  {
+
```

Comment:
nit: Add a comment explaining why this function is empty to indicate it is intentional

Reply:
Done